pkg/vz/vz_driver_darwin.go

```diff
@@ -73,8 +73,9 @@ func (l *LimaVzDriver) Validate() error {
 		"AdditionalDisks",
 		"Audio",
 		"Video",
+		"OS",
 	); len(unknown) > 0 {
-		logrus.Warnf("Ignoring: vmType %s: %+v", *l.Yaml.VMType, unknown)
+		logrus.Warnf("Ignoring following configuration due to the use of vmType %s: %+v", *l.Yaml.VMType, unknown)
```

Comment:
Probably you may want to update all the occurrences of `Ignoring: vmType %s` in this file?

Reply:
Resolved in [1f2c1e2](https://github.com/lima-vm/lima/pull/1769/commits/1f2c1e2ff9363f466d8f743458270f71605301f6).